packages/host/tests/acceptance/code-submode/schema-editor-test.ts

```diff
@@ -740,9 +740,9 @@ module('Acceptance | code submode | schema editor tests', function (hooks) {
 
     assert.ok(
       getMonacoContent().includes(
-        'luckyNumbers = containsMany(BigIntegerCard)',
+        'luckyNumbers = containsMany(BigIntegerField)',
       ),
-      "code editor contains line 'luckyNumbers = containsMany(BigIntegerCard)'",
+      "code editor contains line 'luckyNumbers = containsMany(BigIntegerField)'",
     );
 
     // Field is a card descending from CardDef (cardinality: one)
```

Comment:
is this FieldDef instead of CardDef now?

Reply:
yup! thx